### PR TITLE
updated endpoints

### PR DESCRIPTION
**Pull request by mexicanpepe**
Not sure if this is how the endpoints should look like?

### CODE REVIEW

File: features/LoginSignup/Signin.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const SignIn = ({profile, setProfile}) => {
 
   const sendProfileData = async () => {
     try {
-      const endpoint = '/ENDPOINT';
+      const endpoint = '/api/';
```
Comment (pnilan) [inline features/LoginSignup/Signin.js]: /api/u/users for creating a user, refer to Ahu's nginx host for baseUrl

File: features/LoginSignup/Signin.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ const SignIn = ({profile, setProfile}) => {
       },
     };
 
-    const backendResponse = await axios.get('/ENDPOINT', config);
+    const backendResponse = await axios.get('/api/', config);
```
Comment (pnilan) [inline features/LoginSignup/Signin.js]: GET /api/u/users/:user_id

**Review by pnilan — CHANGES_REQUESTED** [review]

**Review by pnilan — CHANGES_REQUESTED** [review]
Should be HTTP, also there's a forward slash before the URL, also I would recommend referencing a baseURL variable for the IP since the IP will change every time the nginx VM is rebooted

**Review by trilly-con-queso — APPROVED** [review]
lgtm

**Comment by pnilan** [comment]
Merging this

**Pull request merged**